src/view/screens/Profile.tsx

```diff
@@ -163,6 +169,23 @@ function ProfileScreenLoaded({
     ].filter(Boolean) as string[]
   }, [showLikesTab, showFeedsTab, showListsTab])
 
+  let nextIndex = 0
+  const postsIndex = nextIndex++
+  const repliesIndex = nextIndex++
+  const mediaIndex = nextIndex++
+  let likesIndex: number | null = null
+  let feedsIndex: number | null = null
+  let listsIndex: number | null = null
+  if (showLikesTab) {
+    likesIndex = nextIndex++
+  }
+  if (showFeedsTab) {
+    feedsIndex = nextIndex++
+  }
+  if (showListsTab) {
+    listsIndex = nextIndex++
+  }
```

Comment:
This stuff is a bit wonky but I didn't want to change the API contract of the pager, which currently uses the rendered index rather than the passed child index.